dbt/adapters/databricks/relation_configs/tags.py

```diff
@@ -22,7 +22,9 @@ def get_diff(self, other: "TagsConfig") -> Optional["TagsConfig"]:
         for k in other.set_tags.keys():
             if k not in self.set_tags:
                 to_unset.append(k)
-        return TagsConfig(set_tags=self.set_tags, unset_tags=to_unset)
+        if self.set_tags or to_unset:
```

Comment:
I realized while testing that we were potentially issuing alter statements that were unneeded.  This says only return a diff if we are either setting or unsetting some tags.